crates/astria-composer/src/grpc_server_builder.rs

```diff
@@ -0,0 +1,41 @@
+use std::net::SocketAddr;
+
+use astria_eyre::{
+    eyre,
+    eyre::Context,
+};
+use tokio::net::TcpListener;
+use tokio_util::sync::CancellationToken;
+
+use crate::{
+    collectors,
+    executor,
+    grpc::GrpcServer,
+};
+
+pub(crate) struct Builder {
```

Comment:
Move this builder into `grpc.rs` or `mkdir grpc && git mv grpc.rs grpc/mod.rs && grpc_server_builder.rs grpc/builder.rs`